### PR TITLE
Add examples in date_add() & date_diff()

### LINKED ISSUE #1717: Add examples in date_add(), date_diff() documentation
Today the `date_diff()` docs are a bit vague.

- provide some examples (2-3 examples)
- same for `date_add()`

### PR DESCRIPTION
**Pull request by yuokada**
Fix #1717

----

スクリーンショット 2020-02-14 06 41 46

### CODE REVIEW

**Comment by yuokada** [comment]
@findepi Could you take a look if you get a chance?

File: presto-docs/src/main/sphinx/functions/datetime.rst [hunk 1]
```diff
@@ -172,11 +172,19 @@ Unit              Description
 .. function:: date_add(unit, value, timestamp) -> [same as input]
 
     Adds an interval ``value`` of type ``unit`` to ``timestamp``.
-    Subtraction can be performed by using a negative value.
+    Subtraction can be performed by using a negative value::
+
+        SELECT date_add('second', 86400, TIMESTAMP '2020-03-01 00:00:00 UTC'); -- 2020-03-02 00:00:00.000 UTC
+        SELECT date_add('hour',       9, TIMESTAMP '2020-03-01 00:00:00 UTC'); -- 2020-03-01 09:00:00.000 UTC
+        SELECT date_add('day',       -1, TIMESTAMP '2020-03-01 00:00:00 UTC'); -- 2020-02-29 00:00:00.000 UTC
```
Comment (findepi) [inline presto-docs/src/main/sphinx/functions/datetime.rst]: I am thinking we shouldn't vertically align these with spaces. @mosabua ?

File: presto-docs/src/main/sphinx/functions/datetime.rst [hunk 2]
```diff
@@ -172,11 +172,19 @@ Unit              Description
 .. function:: date_add(unit, value, timestamp) -> [same as input]
 
     Adds an interval ``value`` of type ``unit`` to ``timestamp``.
-    Subtraction can be performed by using a negative value.
+    Subtraction can be performed by using a negative value::
+
+        SELECT date_add('second', 86400, TIMESTAMP '2020-03-01 00:00:00 UTC'); -- 2020-03-02 00:00:00.000 UTC
```
Comment (findepi) [inline presto-docs/src/main/sphinx/functions/datetime.rst]: would be good to add small second value (eg 86), to show how seconds/minutes get updated. -- like you did with `hour` example below

File: presto-docs/src/main/sphinx/functions/datetime.rst [hunk 3]
```diff
@@ -172,11 +172,19 @@ Unit              Description
...
+        SELECT date_add('hour',       9, TIMESTAMP '2020-03-01 00:00:00 UTC'); -- 2020-03-01 09:00:00.000 UTC
+        SELECT date_add('day',       -1, TIMESTAMP '2020-03-01 00:00:00 UTC'); -- 2020-02-29 00:00:00.000 UTC
 
 .. function:: date_diff(unit, timestamp1, timestamp2) -> bigint
 
-    Returns ``timestamp2 - timestamp1`` expressed in terms of ``unit``.
+    Returns ``timestamp2 - timestamp1`` expressed in terms of ``unit``::
+
+        SELECT date_diff('second', TIMESTAMP '2020-03-01 00:00:00 UTC', TIMESTAMP '2020-03-02 00:00:00 UTC'); -- 86400
```
Comment (findepi) [inline presto-docs/src/main/sphinx/functions/datetime.rst]: The functions operate on DATE, TIME, TIMESTAMP, TIMESTAMP WITH TIME ZONE.
We don't have to exemplify all of them, but you could eg change the first example to operate on TIMESTAMP (without zone) -- ie remove "UTC" from both operands

**Comment by yuokada** [comment]
Fixed.

**Comment by findepi** [comment]
Merged as b4433452b52bf8370c0910037bcb432a33c1f1d5, thanks!

(squashed)

**Pull request closed without merging**